src/DotNetWorker.Core/CollectionModelBindingData.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker.Core
+{
+    /// <summary>
+    /// A representation of Collection of Microsoft.Azure.WebJobs.ParameterBindingData
+    /// </summary>
+    public abstract class CollectionModelBindingData
+    {
+        /// <summary>
+        /// Gets the array of ModelBindingData
+        /// </summary>
+        public abstract ModelBindingData[] modelBindingDataArray { get; }
```

Comment:
Use PascalCase for public property names.

Reply:
We really need static analysis on build to catch issues like this